### ISSUE TITLE
do you have code example how to do one hot encoding for test data

### ISSUE THREAD

**Issue by Sandy4321** [scikit-learn-contrib/category_encoders#253]
great code thanks
## Expected Behavior
do you have code example how to do one hot encoding for test data

in order to be in synch/match  with train data

something like this

http://fastml.com/how-to-use-pd-dot-get-dummies-with-the-test-set/

**Comment by PaulWestenthanner** [comment]
Please check the example given in the docs
https://github.com/scikit-learn-contrib/category_encoders/blob/master/category_encoders/one_hot.py#L38-L48

For any other test set `X_test` you can just use `enc.transform(X_test)`
